runtime/tests/integration_tests.rs

```diff
@@ -199,6 +201,74 @@ fn test_verify_cmd() {
     );
 }
 
+#[test]
+fn test_hmac_verify_cmd() {
```

Comment:
should this go in hmac.rs?

Or if you want could just omit it. I think running the wycheproof vectors is sufficient. I assume you added to initially test with some known data.

Reply:
Done